Authentication/Login/ViewModel/LoginViewModel.swift

```diff
@@ -22,21 +22,11 @@ public protocol LoginViewModelType {
     var passwordValidationErrors: AnyProperty<[String]> { get }
     var showPassword: MutableProperty<Bool> { get }
     
-    var togglePasswordVisibility: Action<AnyObject, Bool, NoError> { get }
+    var togglePasswordVisibilityCocoaAction: CocoaAction { get }
```

Comment:
To be consistent with the log in action which is a CocoaAction. And at that time, if I'm not mistaken, it was because we wanted to ask the minimum necessary from the user.